app/controllers/concerns/foreman/controller/template_rendering.rb

```diff
@@ -13,7 +13,7 @@ def render_template(template:, type:)
       end
 
       def safe_render(template)
-        renderer = params.delete('force_safemode') ? Foreman::Renderer::SafeModeRenderer : Foreman::Renderer
+        renderer = ActiveModel::Type::Boolean.new.cast(params.delete('force_safemode')) ? Foreman::Renderer::SafeModeRenderer : Foreman::Renderer
```

Comment:
Is there a reason of choosing this over `Foreman::Cast.to_bool`?

Just a side note: it's only used in https://github.com/theforeman/foreman/blob/develop/app/models/compute_resources/foreman/model/vmware.rb#L875, but all other places use `Foreman::Cast`.

Reply:
> Is there a reason of choosing this over `Foreman::Cast.to_bool`?

Only me forgetting that this method existed :sweat_smile: 
Thanks @ofedoren, I have modified the fix accordingly.